src/services/doppler-api-client.double.ts

```diff
@@ -49,35 +49,49 @@ const subscriber = {
   score: 2,
 };
 
-const campaignDeliveryCollection = {
-  items: [
-    {
-      campaignId: 1,
-      campaignName: 'Campaña estacional de primavera',
-      campaignSubject: '¿Como sacarle provecho a la primavera?',
-      deliveryStatus: 'opened',
-      clicksCount: 2,
-    },
-    {
-      campaignId: 2,
-      campaignName: 'Campaña calendario estacional 2019',
-      campaignSubject: 'El calendario estacional 2019 ya está aquí',
-      deliveryStatus: 'notOpened',
-      clicksCount: 23,
-    },
-    {
-      campaignId: 3,
-      campaignName: 'Emms 2019 preveento 1',
-      campaignSubject: 'Ya comienza el dia 2. Accede a las conferencias',
-      deliveryStatus: 'softBounced',
-      clicksCount: 100,
-    },
-  ],
-  currentPage: 0,
-  itemsCount: 3,
-  pagesCount: 1,
+const getDeliveryStatus = (statusNumber: number) => {
+  switch (statusNumber) {
+    case 1:
+      return 'opened';
+    case 2:
+      return 'softBounced';
+    case 3:
+      return 'hardBounced';
+    default:
+      return 'notOpened';
+  }
 };
 
+const campaignDeliveryItems = [...Array(100)].map((_, index) => {
+  return {
+    campaignId: index,
+    campaignName: 'Campaña estacional de primavera',
+    campaignSubject: '¿Como sacarle provecho a la primavera?',
+    deliveryStatus: getDeliveryStatus(Math.round(Math.random() * (5 - 1) + 1)),
+    clicksCount: Math.round(Math.random() * (100 - 1) + 1),
+  };
+});
+
+// const campaignDeliveryItems = [
```

Comment:
Why do you have this commented code? are you going to use it for tests?

Reply:
i saw it, its old! i already deleted